src/sections/SectionHeading/index.js

```diff
@@ -13,10 +13,16 @@ import {
     faSearch,
 } from '@fortawesome/free-solid-svg-icons';
 import { faBell } from '@fortawesome/free-regular-svg-icons';
-import { ShoppingCartWhiteIcon, MessagesBlueWhiteIcon } from '../../components/icons';
+import { ShoppingCartIcon, MessageIcon } from '../../components/icons';
 import Notification from './notification';
+import IconNotification from './iconNotification';
 import './styles.css';
 
+const iconStyles = {
+    height: 12,
+    width: 14,
+};
```

Comment:
Use css, we only keep styles in this way in the app.js.
Also let's put the SectionHeading in the component folder because it is not a section.